tests/system/responses/test_succesful_async_responses.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+
+from umodbus import conf
+from umodbus.client import tcp
+
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture(scope='module', autouse=True)
+def enable_signed_values(request):
+    """ Use signed values when running tests it this module. """
+    tmp = conf.SIGNED_VALUES
+    conf.SIGNED_VALUES = True
+
+    def fin():
+        conf.SIGNED_VALUES = tmp
+
+    request.addfinalizer(fin)
+
+
+@pytest.mark.parametrize('function', [
+    tcp.read_coils,
+    tcp.read_discrete_inputs,
+])
+async def test_response_on_single_bit_value_read_requests(async_tcp_streams, function):
+    """ Validate response of a succesful Read Coils or Read Discrete Inputs
+    request.
+    """
+    slave_id, starting_address, quantity = (1, 0, 10)
```

Comment:
Why not just

```suggestion
    slave_id = 1
    starting_address = 0
    quantity = 10
```

I mean yeah unpacking a set saves you two lines but IMHO it's just slightly less readable

Reply:
This comment applies to every instance of this in your PR

Reply:
I was just trying to be consistent with *test_succesful_responses.py*

Reply:
It should be better now.